src/main/java/it/pagopa/pn/delivery/rest/PnNotificationPriceController.java

```diff
@@ -24,7 +27,20 @@ public PnNotificationPriceController(NotificationPriceService service) {
     @Override
     public ResponseEntity<NotificationPriceResponse> getNotificationPrice(String paTaxId, String noticeNumber) {
         log.info( "Get notification price paTaxId={} noticeNumber={}", paTaxId, noticeNumber );
-        NotificationPriceResponse response = service.getNotificationPrice( paTaxId, noticeNumber );
+        PnAuditLogBuilder auditLogBuilder = new PnAuditLogBuilder();
+        PnAuditLogEvent logEvent = auditLogBuilder
+                .before(PnAuditLogEventType.AUD_NT_REQCOST, "getNotificationPrice")
+                .mdcEntry("paTaxId",paTaxId)
+                .mdcEntry("noticeNumber", noticeNumber)
+                .build();
+        NotificationPriceResponse response = new NotificationPriceResponse();
+        try {
+            response = service.getNotificationPrice( paTaxId, noticeNumber );
+            logEvent.generateSuccess().log();
+        } catch (PnNotFoundException exc) {
+            logEvent.generateFailure("Exception on get notification price: " + exc.getMessage()).log();
+            throw new RuntimeException("Exception on get notification price: " + exc.getMessage(), exc);
```

Comment:
Perché trasformare una PnNotFoundException che dovrebbe essere una exception gestita in una generica RuntimeException?